lnwallet/interface_test.go

```diff
@@ -476,52 +457,6 @@ func testDualFundingReservationWorkflow(miner *rpctest.Harness, wallet *lnwallet
 	if !bytes.Equal(channels[0].FundingOutpoint.Hash[:], fundingSha[:]) {
 		t.Fatalf("channel state not properly saved")
 	}
-
-	// Assert that the channel opens after a single block.
-	lnChan := make(chan *lnwallet.LightningChannel, 1)
-	go func() {
-		openDetails, err := chanReservation.DispatchChan()
-		if err != nil {
-			t.Fatalf("unable to finalize reservation: %v", err)
-		}
-
-		lnChan <- openDetails.Channel
-	}()
-	lnc := assertChannelOpen(t, miner, uint32(numReqConfs), lnChan)
-
-	// Now that the channel is open, execute a cooperative closure of the
```

Comment:
The logic below was removed because the channel can no longer be fully opened from within the wallet package. In order to be fully opened, the FundingManager must wait for the funding transaction to be confirmed. Since the transaction isn't opened by the wallet, I believe the cooperative close test logic needs to be moved to the integration tests.